### PR TITLE
Local dev environment

### PR DESCRIPTION
**Pull request by tom-sherman**
image

### CODE REVIEW

**Comment by damiensedgwick** [comment]
Are there any instructions for getting this running locally? Does it all run off of a single dev database? Give me information please.

File: packages/frontpage/app/(app)/post/[postAuthor]/[postRkey]/[commentAuthor]/[commentRkey]/og-image/route.tsx [hunk 1]
```diff
@@ -61,14 +65,25 @@ export async function GET(
...
+        ) : (
+          <div
+            style={{
+              width: 48,
+              height: 48,
+              borderRadius: "100%",
+              backgroundColor: "#00000073",
+            }}
+          />
+        )}
```
Comment (damiensedgwick) [inline packages/frontpage/app/(app)/post/[postAuthor]/[postRkey]/[commentAuthor]/[commentRkey]/og-image/route.tsx:86]: Is it worth extracting this as I see it is also used in `packages/frontpage/app/(app)/post/[postAuthor]/[postRkey]/og-image/route.tsx`

File: packages/frontpage/local-infra/pds.env
Comment (damiensedgwick) [inline packages/frontpage/local-infra/pds.env:1]: Is this file fine to be committed?

**Review by damiensedgwick — APPROVED** [review]

File: packages/frontpage/local-infra/pds.env
Comment (tom-sherman) [reply packages/frontpage/local-infra/pds.env:1]: Yeah these are fine to be public for a local environment.

**Pull request merged**